spec/support/fixtures.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Fixtures
+  FIXTURE_PATH = Pathname.new(File.expand_path("../fixtures", __dir__))
+  private_constant :FIXTURE_PATH
+
+  def self.path(path)
+    FIXTURE_PATH / path
```

Comment:
Ooh, nice one 😎. I haven't seen `#/` used like this before. For anyone else that is curious, [Pathname#/](https://ruby-doc.org/stdlib-3.0.2/libdoc/pathname/rdoc/Pathname.html#method-i-2B) is an alias for `Pathname#+`.

Reply:
I didn't know about `+`, I thought it was an alias for `#join`. Python uses `/`.